service/test_automatic_run_experiment.py

```diff
@@ -83,12 +83,7 @@ def test_run_requested_experiment(mocked_get_requested_experiments,
         'sqlite3_ossfuzz',
         'systemd_fuzz-link-parser',
         'zlib_zlib_uncompress_fuzzer',
-        'arrow_parquet-arrow-fuzz',
```

Comment:
Why are these no longer expected because they are now bug based?

Reply:
Yes, I added these benchmarks here temporarily [when converting bug-based benchmarks to coverage-based ones](https://github.com/google/fuzzbench/pull/1526/commits/ce36cc916e1f46686995066f4025bab99a13c5de).
I plan to gradually remove them while reviving more of them later, hope you would not mind : )

Reply:
It's problematic that these were made into coverage benchmarks because now coverage experiments have to build these broken benchmarks.